app/bundles/ApiBundle/Translations/en_US/messages.ini

```diff
@@ -25,6 +25,8 @@ mautic.api.client.searchcommand.redirecturi="redirecturi"
 mautic.api.client.thead.publicid="Public Key"
 mautic.api.client.thead.secret="Secret Key"
 mautic.api.config.form.api.enabled="API enabled?"
+mautic.api.config.form.api.basic_auth_enabled="Enable HTTP basic auth(this is insecure)?"
+mautic.api.config.form.api.basic_auth.tooltip="Enable HTTP basic authentication for Mautic's API, use HTTPS otherwise you password can be compromised?"
```

Comment:
Let's change this to `Enable HTTP basic authentication for Mautic's API. It is highly recommended to only use this with secure sites (HTTPS).`